content/dstu2/diagnostic-report.md

```diff
@@ -0,0 +1,26 @@
+---
+title: DiagnosticReport | FHIR DSTU 2 BALLOT API
+---
+
+# Immunization
```

Comment:
Thanks for pointing that out @MaxPhilips Issued a PR to fix this: https://github.com/cerner/fhir.cerner.com/pull/48